### PR TITLE
Wonderful gradients on Dock

### PR DESCRIPTION
**Pull request by PedroHLC**
It's so fucking beautiful, added 1288 lines of beautiful gradients from [uiGradients](http://uigradients.com)

---
This change is [Reviewable](https://reviewable.io/reviews/hackerexperience/heborn/147)

### CODE REVIEW

**Comment by PedroHLC** [comment]
Look how beautiful *-*

**Comment by kress95** [comment]
:joy: :joy: :joy: :ok_hand: :ok_hand: :top: :100:

File: src/Css/Gradients.elm [hunk 1]
```diff
@@ -0,0 +1,1287 @@
...
+
+
+kingYna : Style
+kingYna =
+    gradient3 "1a2a6c" "b21f1f" "fdbb2d"
+
+
+velvetSun : Style
+velvetSun =
+    gradient2 "e1eec3" "f05053"
```
Comment (kress95) [inline src/Css/Gradients.elm]: this is not right :v

I would merge if this file only contained the ones you're using.

File: src/OS/SessionManager/Dock/Style.elm [hunk 2]
```diff
@@ -79,6 +87,14 @@ css =
             , addIco "connmngr" Icon.connMngr
             , addIco "bouncemngr" Icon.bounceMngr
             , addIco "moneymngr" Icon.finance
+            , addGrad "explorer" Gradients.mangoPulp
+            , addGrad "logvw" Gradients.stellar
+            , addGrad "browser" Gradients.pinotNoir
+            , addGrad "taskmngr" Gradients.blurryBeach
+            , addGrad "udb" Gradients.calmDarya
+            , addGrad "connmngr" Gradients.influenza
```
Comment (kress95) [inline src/OS/SessionManager/Dock/Style.elm]: wat, a gradient called flu?

File: src/Css/Gradients.elm (hunk 1 above)
Comment (PedroHLC) [inline src/Css/Gradients.elm]: This is only interpreted when elm-css run, no useless gradients are added on index.css

File: src/OS/SessionManager/Dock/Style.elm (hunk 2 above)
Comment (PedroHLC) [inline src/OS/SessionManager/Dock/Style.elm]: It seems like 😆

**Comment by kress95** [comment]
Reviewed 1 of 66 files at r1.
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/Css/Gradients.elm, line 1287 at r1](https://reviewable.io:443/reviews/hackerexperience/heborn/147#-KoJTLcWWeMWsZkBfFfz-r1-1287:-KoO6tadlBiVR8bIICZZ:b-qws919) ([raw file](https://github.com/hackerexperience/heborn/blob/f54d1fa16e5b445878715b370239a8576f61b55c/src/Css/Gradients.elm#L1287)):*
<details><summary><i>Previously, PedroHLC (Pedro) wrote…</i></summary><blockquote>

This is only interpreted when elm-css run, no useless gradients are added on index.css
</blockquote></details>

Alright, I'll accept it, but I don't have any stomach to merge it :v
Either you/renato should do it :joy:

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/hackerexperience/heborn/147)*

**Comment by kress95** [comment]
Reviewed 1 of 66 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/hackerexperience/heborn/147)*

**Pull request closed without merging**